pkg/manifests/config.go

```diff
@@ -31,23 +31,28 @@ type Config struct {
 	RemoteWrite bool                  `json:"-"`
 	Platform    configv1.PlatformType `json:"-"`
 
-	PrometheusOperatorConfig             *PrometheusOperatorConfig `json:"prometheusOperator"`
-	PrometheusOperatorUserWorkloadConfig *PrometheusOperatorConfig `json:"prometheusOperatorUserWorkload"`
-
-	PrometheusK8sConfig          *PrometheusK8sConfig `json:"prometheusK8s"`
-	PrometheusUserWorkloadConfig *PrometheusK8sConfig `json:"prometheusUserWorkload"`
+	ClusterMonitoringConfiguration *ClusterMonitoringConfiguration `json:"-"`
+	UserWorkloadConfiguration      *UserWorkloadConfiguration      `json:"-"`
```

Comment:
Having both `UserWorkloadConfiguration` and `UserWorkloadConfig` is confusing.

Reply:
UserWorkloadConfig?

Reply:
We have a `UserWorkloadConfig` struct:
https://github.com/openshift/cluster-monitoring-operator/blob/882dc1b7310edd7981fab2aff76fdc30672a470c/pkg/manifests/config.go#L157-L159

And another `UserWorkloadConfiguration` struct:
https://github.com/openshift/cluster-monitoring-operator/blob/882dc1b7310edd7981fab2aff76fdc30672a470c/pkg/manifests/config.go#L381-L385

The names are close which makes it confusing for me when reviewing the code.

Reply:
I see, makes sense! Any suggestions for new or old name?